apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -91,6 +91,10 @@ function WelcomeTourCard( {
 }
 
 function CardNavigation( { cardIndex, lastCardIndex, onDismiss, setCurrentCardIndex } ) {
+	// These are defined on their own lines because of a minification issue.
+	// __('translations') do not always work correctly when used inside of ternary statements.
+	const startTourLabel = __( 'Start Tour', 'full-site-editing' );
```

Comment:
To me, this is a nice refactor in general. Nice organization have the strings defined here. :) (In addition to solving the minification problem.)